packages/siopv2-oid4vp-rp-rest-api/src/siop-api-functions.ts

```diff
@@ -88,8 +89,15 @@ export function verifyAuthResponseSIOPv2Endpoint(
         // const credentialSubject = wrappedPresentation.presentation.verifiableCredential[0]?.credential?.credentialSubject
         // console.log(JSON.stringify(credentialSubject, null, 2))
         console.log('PRESENTATION:' + JSON.stringify(wrappedPresentation.presentation, null, 2))
-        const responseRedirectURI = await context.agent.siopGetRedirectURI({ correlationId, definitionId, state: verifiedResponse.state })
         response.statusCode = 200
+
+        const authorizationChallengeValidationResponse: AuthorizationChallengeValidationResponse = { presentation_during_issuance_session: verifiedResponse.correlationId }
+        if (authorizationResponse.is_first_party) {
```

Comment:
This looks spec incompliant and thus incompatible with any other wallet. Should be derived from metadata and/or certain attributes in responses instead of having our own prop in the response

Reply:
created a ticket for this https://sphereon.atlassian.net/browse/MWALL-721